src/main/kotlin/com/spectralogic/bp/bench/cli/MemoryBuffer.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * ****************************************************************************
+ *   Copyright 2014-2019 Spectra Logic Corporation. All Rights Reserved.
+ * ***************************************************************************
+ */
+package com.spectralogic.bp.bench.cli
+
+import com.spectralogic.ds3client.helpers.Ds3ClientHelpers
+import java.nio.ByteBuffer
+import java.nio.channels.SeekableByteChannel
+import kotlin.random.Random
+
+class MemoryObjectChannelBuilder(private val bufferSize: Int, private val sizeOfFiles: Long) :
+    Ds3ClientHelpers.ObjectChannelBuilder {
+
+    override fun buildChannel(key: String): SeekableByteChannel = DevNullByteChannel(bufferSize, sizeOfFiles)
+
+    private inner class DevNullByteChannel(private val bufferSize: Int, private val limit: Long) : SeekableByteChannel {
+        private val backingArray: ByteArray = Random.nextBytes(bufferSize)
```

Comment:
Should we be accessing this statically? Or passing it in?  Additionally should we allow them to specify a seed so they can generate the same data sequence?